openhtf/exe/test_state.py

```diff
@@ -63,6 +64,7 @@ def __init__(self, test_data, plug_manager, dut_id, station_id):
     self.record = test_record.TestRecord(
         dut_id=dut_id, station_id=station_id, code_info=test_data.code_info,
         metadata=test_data.metadata)
+    self.record.metadata['config'] = conf._asdict()
```

Comment:
can you initialize it when the metadata gets initialized, i believe upon creation of the test data object.